src/coreclr/jit/codegenriscv64.cpp

```diff
@@ -1014,6 +1014,111 @@ void CodeGen::genCodeForIncSaturate(GenTree* tree)
     genProduceReg(tree);
 }
 
+// Generate code for multiplications by power of 2.
+void CodeGen::genCodeForMul(GenTreeOp* treeNode)
```

Comment:
Why is this needed instead of it hitting the standard morph optimization which is meant to do the same?

Reply:
Oops. I think this is good for practicing newcomer.
I missed checking the opt. Sorry, it is my fault. 
I close it.